client/actions/SignUpAction.js

```diff
@@ -11,7 +11,7 @@ import authAction from './authAction';
  */
 export default function userSignupRequest(userData) {
   return (dispatch) => {
-    axios.post('http://127.0.0.1:3300/api/user/signup', userData)
+    axios.post('http://127.0.0.1:3300/api/v1/user/signup', userData)

```

Comment:
I think you could have defined a baseURL for all your axios call, so instead of having to write the full url it could just be `v1/user/signup` or `user/signup` as opposed `http://127.0.0.1:3300/api/v1/user/signug`